test/cobalt/evaluators/format_check.py

```
@@ -0,0 +1,216 @@
+        A dict with score and reasoning.
+    """
+    stripped = output.strip()
+
+    # Try to extract JSON from code blocks
+    code_block_match = re.search(r'```(?:json)?\s*\n(.*?)\n```', stripped, re.DOTALL)
+    if code_block_match:
+        stripped = code_block_match.group(1).strip()
+
+    try:
+        json.loads(stripped)
+        return {'score': 1.0, 'reasoning': 'Output is valid JSON.'}
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)